src/lambda/commands/deleteLambda.ts

```diff
@@ -5,37 +5,63 @@
 
 'use strict'
 
-import * as nls from 'vscode-nls'
-const localize = nls.loadMessageBundle()
+import { OutputChannel, window } from 'vscode'
+import { loadMessageBundle } from 'vscode-nls'
+const localize = loadMessageBundle()
 
 import { LambdaClient } from '../../shared/clients/lambdaClient'
-import { ext } from '../../shared/extensionGlobals'
 import { StandaloneFunctionNode } from '../explorer/standaloneNodes'
 
-export async function deleteLambda(
-    node: StandaloneFunctionNode,
-    refresh: () => void
-): Promise<void> {
-    const client: LambdaClient = ext.toolkitClientBuilder.createLambdaClient(node.regionCode)
+export async function deleteLambda({
```

Comment:
When we chatted about destructured-objects-as-parameters, I didn't realize how much boilerplate it would add in TypeScript. Let's keep our codebase consistent for now--please switch to regular parameters.

Reply:
Removed "boiler plate" of repeated parameter names in favor of a single restParams parameter. Long lists of arguments to a function/constructor has been considered poor practice in many languages for a long time. This is just as true in TypeScript as it is in JavaScript and Java.

https://medium.com/@janakachathuranga/coding-best-practices-javascript-9a5f397d701b

Look at this example: 
```
                new AWSCommandTreeNode(
                    undefined,
                    localize('AWS.explorerNode.signIn', 'Connect to AWS...'),
                    'aws.login',
                    undefined,
                    localize('AWS.explorerNode.signIn.tooltip', 'Connect to AWS using a credential profile'))
```
What do each of those arguments represent? You may know but the typical reader does not.

By passing a single typed object, the intent is obvious:
```
            async (node: StandaloneFunctionNode) => await deleteLambda({
                node,
                lambdaClient: ext.toolkitClientBuilder.createLambdaClient(node.regionCode),
                outputChannel: ext.lambdaOutputChannel,
                onRefresh: () => this.refresh(node.parent)
})
```
I also feel this CR is a reasonable place for this change. If a function has one argument, that's fine, two maybe, but when a CR add a 3rd or 4th it should change the function to take a single typed object as an argument.